src/server/coprocessor.rs

```diff
@@ -199,12 +199,12 @@ impl TiDbEndPoint {
                 return;
             }
         };
-        debug!("[TIME_SNAPSHOT] {:?}", ts.elapsed());
+        metric_time!("COPR_SNAPSHOT", ts.elapsed());
```

Comment:
I think we should use "copr" tag first like "raftstore", so here may be "corp.snapshot"